LoopKitUI/Views/MultiButtonTableFooterView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  MultiButtonTableFooterView.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+
+public class MultiButtonTableFooterView: UIView {
+
+    public let primaryButton: TableFooterButton?
+
+    public let secondaryButton: TableFooterButton?
+
+    public let destructiveButton: TableFooterButton?
+
+    public let cancelButton: TableFooterButton?
+
+    public init(frame: CGRect,
+                primaryButton: TableFooterButton? = nil,
+                secondaryButton: TableFooterButton? = nil,
+                destructiveButton: TableFooterButton? = nil,
+                cancelButton: TableFooterButton? = nil) {
+        self.primaryButton = primaryButton
+        self.secondaryButton = secondaryButton
+        self.destructiveButton = destructiveButton
+        self.cancelButton = cancelButton
+        
+        super.init(frame: frame)
+        autoresizingMask = [.flexibleWidth]
+
+        primaryButton?.setStyle(forColor: .loopSelectable)
+        secondaryButton?.setStyle(forColor: .loopSelectable, borderOnly: true)
+        destructiveButton?.setStyle(forColor: .loopDestructive)
+        cancelButton?.setStyle(forColor: .loopDestructive, borderOnly: true)
+
+        var buttons: [TableFooterButton] = []
```

Comment:
[nit] Consider using an array and `compactMap`.

Reply:
nice!